src/components/panel/panel.spec.js

```diff
@@ -1450,6 +1633,41 @@ describe('$mdPanel', function() {
     attachedElements.push(element);
   }
 
+  function clickOutsideToClosePanel() {
+    if (!panelRef) {
+      return;
+    }
+
+    var container = panelRef._panelContainer;
+
+    container.triggerHandler({
+      type: 'mousedown',
+      target: container[0]
+    });
+
+    container.triggerHandler({
+      type: 'mouseup',
+      target: container[0]
+    });
+
+    flushPanel();
+  }
+
+  function escapeToClosePanel() {
```

Comment:
nit: this function name should indicate that an event will take place. It really has nothing to do with closing the panel. Perhaps, `pressEscape`?